heapster.go

```diff
@@ -40,7 +40,7 @@ var (
 	argSinkFrequency   = flag.Duration("sink_frequency", 10*time.Second, "Frequency at which data will be pushed to sinks")
 	argCacheDuration   = flag.Duration("cache_duration", 5*time.Minute, "The total duration of the historical data that will be cached by heapster.")
 	argUseModel        = flag.Bool("use_model", false, "When true, the internal model representation will be used")
-	argModelResolution = flag.Duration("model_resolution", 2*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
+	argModelResolution = flag.Duration("model_resolution", 1*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
```

Comment:
Model updates should ideally occur every 1 - 3 resolutions. We cannot update more often than the model resolution with the current implementation.

Moreover, the update operation should not be an exact multiple of resolution, to be able to serve more accurate values at the latest minute. For that reason, I set the update frequency to min(cacheDuration, 2 * ModelResolution + 25 sec). This minimizes the number of Update operations that occur at the beginning of the latest resolution, which could decrease accuracy for metrics that are extracted on the very last minute.

I can open an issue with the required steps, if more often updates are required.